src/app/[locale]/(app)/content/_components/PartnersFooter.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+import { CldImage } from "next-cloudinary";
+import { useTranslations } from "next-intl";
+import { usePathname } from "next/navigation";
```

Comment:
To fix linters and the builds:
```suggestion
import { usePathname } from "@/i18n/routing";
```